integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/ItMiiSampleHelper.java

```diff
@@ -228,7 +230,7 @@ public void execTestScriptAndAssertSuccess(DomainType domainType,
 
         boolean success =
             result != null
-                && result.exitValue() == 0
+                && (result.exitValue() == 0 || cleaupFailed(result))
```

Comment:
The script could fail due to other reasons than the kill in combination with a kill failure, so this check would assume success even if something other than the kill went wrong.

Reply:
The validation also checks that the script standard output contains string "Finished without errors". IMO, the chances of test script finishing with message "Finished without errors" and other reasons in combination with a kill failure is very rare if any. If the test fails again, we can evaluate and add other checks.